docs/tutorials.rst

```diff
@@ -103,6 +103,156 @@ Which should print:
     hello, world
 
 
+Install packages slices into a ROCK
+===================================
+
+In this tutorial you will create a lean ROCK that contains a fully functional OpenSSL installation, and you will verify
```

Comment:
```suggestion
In this tutorial, you will create a lean ROCK that contains a fully functional OpenSSL installation, and you will verify
```

Reply:
Do you happen to have a reference for comma usage in Canonical docs? My instinct (or personal style I guess) is that this comma breaks the pace of the sentence but if it's recommended I'll add it

Reply:
I use Grammarly (English UK) for spell-checking. In this tutorial you will create a lean ROCK that contains a fully functional OpenSSL installation, and you will verify:

Quoting:

```
Add a comma
An introductory word (Unfortunately), phrase (Before the party), or clause (If you can join us) sets the stage for the sentence. Introductory clauses and long phrases must always be followed by a comma, while single words and short phrases may or may not. Use a comma when the phrase is a verbal phrase (Running away), a non-essential appositive, or an absolute phrase (The audience waiting). Also use a comma when a pause is necessary or the lack of a comma will cause confusion.

INCORRECT:
Prior to the scandal the governor’s approval rating was high.
CORRECT:
Prior to the scandal, the governor’s approval rating was high.
INCORRECT:
At the end of the day money is just money.
CORRECT:
At the end of the day, money is just money.
A writer may use discretion to determine whether a comma is needed. Here is an example which creates a subtle difference.
CORRECT:
Of course I am happy.
CORRECT:
Of course, I am happy.
```

Reply:
Thanks! updated